web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,189 @@
+# Create A Virtual Machine
+
+## Launching the Wizard
+
+![Create VM](img/create-vm.png)
+
+When no virtual machines exist, the Virtual Machines section of the UI will include quick access to the Create Virtual Machine wizard in its empty state.
+
+
+## Step 1: Basic
+
+![Basic step](img/Step-1-basic-template.png)
+
+The wizard has 5 steps: Basic, Networking, Storage, Advanced, and Review.
+
+The first (alphabetical) template is selected by default if available. Provision Source, Operating System, Flavor, and Workload Profile are pre-filled by the template, but only Flavor can be modified between either the predefined option or custom. The user can switch to a different template, or choose the top-most “None” option to not use a template at all.
+
+The “Start virtual machine on creation” checkbox will do exactly that. The “Create new template from configuration” checkbox will be enabled if the user changes the Provision Source, OS, Flavor, or Workload Profile fields.
+
+Once a name is provided the user can create the VM via the "create virtual machine" action. Clicking this button will immediately bring them to the “Review” step and initiate the creation/import process.
+
+If the user modifies the OS, Flavor, or Workload profile, the “Create new template from configuration” checkbox will no longer be disabled.
+
+## States
+### No Namespace State
+
+![No namespace](img/Step-1-basic-namespace.png)
+
+If the user initializes this wizard while viewing “All Projects” without a specific Namespace selected, a new required Namespace field should appear above the template dropdown.
+
+### No Template State
+
+![No template](img/Step-1-basic-no-template-chosen.png)
+
+
+If no templates are available the Templates field should be disabled with inner text of “No templates available”.
+
+Because a Provision Source has not been chosen, the OS, Flavor, and Workload Profile dropdowns should be disabled.
+
+As soon as the OS, Flavor, or Workload profile fields are modified, the “Create new template from configuration” checkbox will no longer be disabled.
+
+## template
```

Comment:
Nit: Should be capital T in Template

Reply:
@andybraren ya I moved it up to align with the field order. I don't feel strongly about it though, if we stick with this approach I guess Namespace should be first. WDYT?

Reply:
By this I mean:

```
## States
### No Namespace State
### No Template State
### template <- the section in question, isn't really a "state"

## Provision Sources
### <- "Template" should go somewhere around here as a Provision Source
### Import
etc.
```

Reply:
I've updated this to align with the field order and added that to the sections. WDYT?